network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -18,14 +18,15 @@
     col-md-12
   {% endif %}"
 >
-  <div class="row">
+  <div class="row"> 
```

Comment:
Nitpick: get rid of the space here. Your editor should be doing this automatically though

Reply:
not sure how it missed that, but done :shipit: